Source/JavaScriptCore/heap/Heap.cpp

```diff
@@ -136,6 +136,14 @@ size_t proportionalHeapSize(size_t heapSize, size_t ramSize)
     if (VM::isInMiniMode())
         return Options::miniVMHeapGrowthFactor() * heapSize;
 
+    // Use new heuristic function for machines >= 16GB RAM.
+    // https://www.mathway.com/ja/Algebra?asciimath=2%20*%20e%5E(-1%20*%20x)%20%2B%201%20%3Dy
```

Comment:
I think this is better?
```
https://www.mathway.com/Algebra?asciimath=2%20*%20e%5E(-1%20*%20x)%20%2B%201%20%3Dy
```
without `ja`.

Reply:
Oops, lol, dropped.